C/services/south/include/ingest.h

```diff
@@ -47,6 +47,22 @@
 
 class IngestRate;
 
+// Enum for service buffering type
+enum class ServiceBufferingType {
+	UNLIMITED,
+	LIMITED
+};
+
+// Enum for discard policy
+enum class DiscardPolicy {
+	DISCARD_OLDEST,
+	REDUCE_FIDELITY,
+	DISCARD_NEWEST
+};
+
+#define SERVICE_BUFFER_BUFFER_TYPE_DEFAULT 	ServiceBufferingType::UNLIMITED
+#define SERVICE_BUFFER_DISCARD_POLICY_DEFAULT 	DiscardPolicy::REDUCE_FIDELITY
```

Comment:
https://github.com/fledge-iot/fledge/blob/481bda78939e3c291581bfc9c2c6964898cb3c27/python/fledge/services/core/server.py#L347

The default Discard Policy must be uniform across all references.